### ISSUE TITLE
Is the project still active?

### ISSUE THREAD

**Issue by NemoN** [netzsinus/defluxio-hardware#3]
Even the forums seems to be down?

**Comment by fti7** [comment]
Doesn't seem so :-( unfortunately....
Did you find any other Resources/Projects for maybe updated hardware @NemoN ?
@gonium ping

**Comment by corbolais** [comment]
ping

**Comment by NemoN** [comment]
pong ;-)

Has anyone built the hardware lately?
Maybe with a pcb layout (https://pcbs.io/)

**Comment by NemoN** [comment]
just uploaded the Eagle file to pcbs.io
4 pieces of the pcb will costs $47.48
(anyone from germany is interessted?)

**Comment by corbolais** [comment]
@NemoN ~50USD seems on the pricey side of PCBs.
The schematic is easily setup on veraboard (~50cent) or similar.

**Comment by NemoN** [comment]
@corbolais the pcb is HUGE. and this is the price for 4 of them  i will share the project.

**Comment by NemoN** [comment]
@corbolais i'm open for cheaper alternatives/design

**Comment by corbolais** [comment]
> @corbolais the pcb is HUGE. and this is the price for 4 of them i will share the project.

How "HUGE" can it be with one handfull of parts? See [schematic](https://github.com/netzsinus/defluxio-hardware/blob/master/hardware/pics/schaltplan-v0.2.jpg).
It's mostly empty: https://www.altium.com/viewer?token=RCPrv1w3Z06RtPVzClU0qLAb . 
That's alot of cash for empty FR4.
Cheers.

**Comment by NemoN** [comment]
@corbolais yes the mostly empty, but the dimensions are large: https://PCBs.io/share/zabeJ

**Comment by gillhofer** [comment]
Recently I started to Update the HumPi Project. 
https://github.com/gillhofer/HumPi
